### PR TITLE
Bug/map incident count2

### PR DESCRIPTION
**Pull request by austin-carman**
# Description

This pull request fixes a bug with the map feature on the home page of the application. The map's incident clusters were not increasing the count after a new incident was approved through the admin dashboard. This PR now allows for the admin to approve an incident that will be mapped according to latitude and longitude of the incident and increment the count of the corresponding map cluster for that area. 

Contributors:
Austin Carman - @austin-carman 
Onaje (Naj) Oladuwa - @ooladuwa 
Ahmed Serag - @ahmedseragcodes 

<a href="https://www.loom.com/share/acd86946d11b425bbcb7de755ef0f1bb">
    <p>bug/map-incident-count2 - Watch Video</p>
    
  </a>

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] `npm test`

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by ahmedseragcodes — APPROVED** [review]
Changes make sense and are necessary for getting mapquest API working properly, no changes to variables or properties that echo out of scope and create too much risk for bugs, also collaborated with Austin on this, approved.

**Review by ooladuwa — APPROVED** [review]
All changes were logical and necessary. Map now increments after incident is approved from the admin dashboard(after full page refresh). I worked with Austin on this bug.

File: src/utils/DashboardHelperFunctions.js
Comment (paulstgermain) [inline src/utils/DashboardHelperFunctions.js:79]: This extra error handling for an edge case where the user makes an error and forgets to fill out both city & state was a nice touch!

File: src/utils/DashboardHelperFunctions.js
Comment (paulstgermain) [inline src/utils/DashboardHelperFunctions.js:89]: Also love the simplified `get` request by way of passing our completed URL to a variable, then that variable to the request!

File: src/utils/DashboardHelperFunctions.js
Comment (paulstgermain) [inline src/utils/DashboardHelperFunctions.js:95]: Always nice to actually _see_ the errors our requests throw!

**Review by paulstgermain — APPROVED** [review]
## Video Walkthrough Feedback

- You lead off with a pretty strong explanation of the problem this PR seeks to solve - this could only be improved by preparing examples to *show* the problem as well as talk about it.
- Speaking of leading with examples, what you do really well here is walk through the code and the actual flow of data moving from point A to point B, making clear the process by which we pull the latitude and longitude of every location in each approved incident moving forward and send it to the Mapquest API. I would have loved a walkthrough of the `getLatAndLong` function to complete the loop, but this is otherwise sufficiently explained!
- This is wrapped up by a fantastic example of the result of the work put into this PR, clearly demonstrating that the code works! I also appreciate that you mentioned the need for refreshing the page being a potential problem for our users - this is something we'll want to look into as this app nears shipping, and it's well documented here.

## Code Review

This is an immensely important PR for such a small amount of code actually changed - in making the modifications you all did here you've solved a major problem with the app and brought us closer to the vision laid out by our stakeholders and previous contributors alike!

The code at hand is simple, clean, and effective - I see no problems here, and so I'm approving this PR.

As a note, however, we'll want to revisit the fact that the admin user will need to refresh the app to see the changes reflected on the home page's map - maybe we could use React's `history` object to trigger a move/reload back to the home page, or otherwise implement a refresh on the incidents page!

**Pull request merged**
